<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AshuraWind</class>
 <widget class="QMainWindow" name="AshuraWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto Bk</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TextChanger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Tekst zmian</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QCheckBox" name="qcheck_save">
             <property name="text">
              <string>S</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="qexpression">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>(?&lt;from&gt;\\w+) to (?&lt;to&gt;\\w+)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="qgenerate">
             <property name="font">
              <font>
               <family>Roboto Bk</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Generuj zmiany</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="qmodifier">
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Linii:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="qspin_line_count">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Słów:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="qspin_word_count">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Tabela zmian</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QTableWidget" name="qtablechanges">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Szukana</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Zmień na</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Bk</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="font">
     <font>
      <family>Roboto Bk</family>
     </font>
    </property>
    <property name="title">
     <string>Plik</string>
    </property>
    <widget class="QMenu" name="menuOstatnie_teksty_zmian">
     <property name="title">
      <string>Ostatnie teksty zmian-&gt;</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuOstatnie_teksty_wej_ciowe">
     <property name="title">
      <string>Ostatnie teksty wejściowe-&gt;</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuOstatnio_zapisane_pliki">
     <property name="title">
      <string>Ostatnio zapisane pliki-&gt;</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionWczytaj_tekst_zmian"/>
    <addaction name="actionWczytaj_tekst_wej_ciowy"/>
    <addaction name="actionZapisz_tekst_wyj_ciowy"/>
    <addaction name="separator"/>
    <addaction name="menuOstatnie_teksty_zmian"/>
    <addaction name="menuOstatnie_teksty_wej_ciowe"/>
    <addaction name="menuOstatnio_zapisane_pliki"/>
    <addaction name="separator"/>
    <addaction name="actionWyjd"/>
   </widget>
   <widget class="QMenu" name="menuSzablony">
    <property name="title">
     <string>Szablony</string>
    </property>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuSzablony"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Tekst wejściowy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="qtextinput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Tekst wyjściowy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="qtextoutput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionWczytaj_tekst_zmian">
   <property name="text">
    <string>Wczytaj tekst zmian</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionWczytaj_tekst_wej_ciowy">
   <property name="text">
    <string>Wczytaj tekst wejściowy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionZapisz_tekst_wyj_ciowy">
   <property name="text">
    <string>Zapisz tekst wyjściowy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionWyjd">
   <property name="text">
    <string>Wyjdź</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionWyjd</sender>
   <signal>triggered()</signal>
   <receiver>AshuraWind</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
